sources/web/datalab/polymer/modules/ApiManager.ts

```diff
@@ -143,6 +144,18 @@ class ApiManager {
   static xsrfToken = '';
 
   /**
+   * Current connection status. Set to the last connection status.
+   */
+  static isConnected = true;
+
+  /**
+   * Handlers for connected/disconnected status. These will be called when the
+   * isConnected property changes value.
+   */
+  static connectedHandler: Function;
```

Comment:
Added signature.
I'd love to use an event target and listeners here, but I don't know how without using a DOM element. Let me know if you have any ideas.